Add `git diff` to publishing steps in readme (#646)
zeit/now-builders: README.md

yarn publish-canary
```

> For the Stable Channel, you must :

Reviewer: ```suggestion
For the Stable Channel, you must do the following:
```

- Cherry pick each commit from canary to master

> - Verify that you are *in-sync* with canary with `git diff`

Reviewer: ```suggestion
- Verify that you are *in-sync* with canary (with the exception of the `version` line in `package.json`)
```

git cherry-pick <PR503_COMMIT_SHA>
git cherry-pick <PR504_COMMIT_SHA>
# ... etc ...

> git diff canary

Reviewer: ```suggestion
git diff origin/canary
```
